### ISSUE TITLE
Typings: Cypress is not declared correctly

### ISSUE THREAD

**Issue by vince91** [cypress-io/cypress#1006]
- Cypress Version: 1.1.3

### Is this a Feature or Bug?

Bug

### Current behavior:

`Cypress` is not declared correctly inside the typings. For instance it does not contain `Cookies`.

![image](https://user-images.githubusercontent.com/6903139/33567884-e6f83ce8-d924-11e7-9b31-c9cb6c9c9239.png)

This is because `Cookies` is defined inside the interface `Core` whereas it should be defined inside the interface `Cypress`, as it is the latter one which is declared in the last line of `index.d.ts`.

What is inside the interface `Core` should be inside the interface `Cypress`.

It does not prevent cypress to work. It's just a linting issue.

#### Test code:

```js
describe('Example', () => {
    before(() => {
        Cypress.Cookies.preserveOnce('cookie_name');
        cy.visit('http://localhost');
    });
});
```

**Comment by bahmutov** [comment]
Yeah we are not using typescript enough to detect it yet. Any pull requests with corrected typings in cli/index.d.ts are welcome!

**Comment by vince91** [comment]
All right, I will post a PR tomorrow.

**Comment by brian-mann** [comment]
Any chance you could do today? We might have another release tonight.

**Comment by marceloemanoel** [comment]
Apparently, it didn't get into the release. Do you have a date for a new version? I'm  just waiting for this to remove @types/cypress and our typings in favor of cypress' one.

**Comment by bahmutov** [comment]
Version 1.1.3 has been released - see if it has types

**Comment by marceloemanoel** [comment]
Hi, @bahmutov thanks for the quick reply. Version 1.1.3 does have the typings, but they have the initial problem pointed in the thread. Also, the commands interface is outdated, I do have a lot of custom commands were updated with the latest change, they would break if I use version 1.1.3. This commit was included later https://github.com/cypress-io/cypress/commit/532cf38208b1a4bd79eb5c18d340303a584ef469#diff-7c9ab0d4f69877664cbcf4e39f2b7eee

**Comment by bahmutov** [comment]
Yes, the merged pull request with typings has not been published yet (hope to publish 1.1.4 soon though)

**Comment by brian-mann** [comment]
Released in [`1.1.4`](https://docs.cypress.io/guides/references/changelog.html#1-1-4).

**Comment by marceloemanoel** [comment]
Thanks!
